lms/djangoapps/certificates/models.py

```diff
@@ -508,25 +510,25 @@ def certificate_status(generated_certificate):
     This returns a dictionary with a key for status, and other information.
     The status is one of the following:
 
-    unavailable  - No entry for this student--if they are actually in
-                   the course, they probably have not been graded for
-                   certificate generation yet.
-    generating   - A request has been made to generate a certificate,
-                   but it has not been generated yet.
-    regenerating - A request has been made to regenerate a certificate,
-                   but it has not been generated yet.
-    deleting     - A request has been made to delete a certificate.
-
-    deleted      - The certificate has been deleted.
-    downloadable - The certificate is available for download.
-    notpassing   - The student was graded but is not passing
-    restricted   - The student is on the restricted embargo list and
-                   should not be issued a certificate. This will
-                   be set if allow_certificate is set to False in
-                   the userprofile table
-    unverified   - The student is in verified enrollment track and
-                   the student did not have their identity verified,
-                   even though they should be eligible for the cert otherwise.
+    unavailable     - No entry for this student--if they are actually in
```

Comment:
Not sure what's going on with the spacing here. Not a big deal either way but it looks like some tabs got added in here?